src/assets/erc20/rsk-tokens.json

```diff
@@ -49,5 +49,40 @@
     "decimals": 18,
     "contractAddress": "0x055a902303746382fbb7d18f6ae0df56efdc5213",
     "color": "#4D9CDC"
+  },
+  "BDX": {
+    "name": "Blindex",
+    "code": "BDX",
+    "decimals": 18,
+    "contractAddress": "0x6542a10e68ceac1fa0641ec0d799a7492795aac1",
+    "color": "#ED6C0F"
+  },
+  "BDUS": {
+    "name": "Blindex USD",
+    "code": "BDUS",
+    "decimals": 18,
+    "contractAddress": "0xb450ff06d950efa9a9c0ad63790c51971c1be885",
+    "color": "#ED6C0F"
+  },
+  "BDEU": {
+    "name": "Blindex EUR",
+    "code": "BDEU",
+    "decimals": 18,
+    "contractAddress": "0x99ac494badd0cba26143bd423e39a088591c7b09",
+    "color": "#ED6C0F"
+  },
+  "XUSD": {
+    "name": "Babelfish USD",
+    "code": "XUSD",
+    "decimals": 18,
+    "contractAddress": "0xb5999795be0ebb5bab23144aa5fd6a02d080299f",
+    "color": "#7D7E7D"
+  },
+  "ETHs": {
+    "name": "Rootstock ETH",
+    "code": "ETHs",
+    "decimals": 18,
+    "contractAddress": "0x1d931bf8656d795e50ef6d639562c5bd8ac2b78f",
+    "color": "#333333"
```

Comment:
@MichalSzreder let's add our new tokens: bXAU and bGBP on both PRs

Reply:
If some of these tokens is listed on coingecko, can you add the property 
"coinGeckoId". 

If coinGeckoId is not available, because  the tokens are not listed on coingecko, you can also use 
"matchingAsset":  - (it links the price of your token to the price of some other token. e.g you can link the price of your stable $ token to USDC)

example for other token 
  "PUSDC": {
    "name": "Polygon USD Coin",
    "code": "USDC",
    "decimals": 6,
    "contractAddress": "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174",
    "color": "#5b31b9",
    "coinGeckoId": "usd-coin",
    "matchingAsset": "USDC"
  },

Reply:
Hmm, I'm not sure if _matchingAsset_ is what we want since the prices can, and will differ slightly, which could make a difference for bigger swaps and confuse users. @tkeidar WDYT?

Reply:
Hmm, I'm not sure if _matchingAsset_ is what we want since the prices can, and will differ slightly, which could make a difference for bigger swaps and confuse users. @tkeidar WDYT?

Reply:
I agree @MichalSzreder.

@tima-t can you elaborate on what this price is used for?
basically, the actual token price is determined by our AMM, which is a Uniswap V2, can we use it?

Reply:
As far as I can see it is used only for displaying token price in USD

Reply:
Yes, coingeckoId is used to display the price in USD. 

I was trying to say that if the price of your BUSD and XUSD is always equal to  approx. 1 dollar. 

You can link your tokens to the price feed of coingeckoId and the price of USD.  That way it will be more user friendly. 

This will work in case the price of you token is stable and the price range  is between than 0.99 to 1.01. 
It is only for informational purposes (this usd price), but we do not want to show wrong price. 

Same thing I am proposing for  ETHs if the price of ETHs is always equal to the price of ETH

Reply:
@MichalSzreder For XUSD we can use the price of USDT and ETH for ETHs.
The other tokens we will keep empty for now

Reply:
The tokens are stable but the price is not always between that range for BDUS nor USDC @tkeidar WDYT?

Reply:
I have added these for XUSD and ETHs